### DISCUSSION TITLE
Welcome to Pretty Cool Fuzzy Guesser (PCFG) Discussions!

### DISCUSSION THREAD

**Discussion by lakiw** [lakiw/pcfg_cracker#23] [Announcements]
## 👋 Welcome!
I'll be honest, I have no idea what github discussions are, but I figure this might enable a better way for people to ask questions and talk about how the PCFG tool is used without having to open a GIT Issue. I hope that everyone can:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please don't be a jerk and treat people with respect. Other than that, feel free to ask questions or share stories.

**Comment by Legoclones** [comment]
I'm hoping to eventually take this project and transform it into an open source version of the TarGuess models.

**Reply by lakiw** [reply]
Let me know how I can help! I've been toying around for years with the idea of modifying existing rulesets/grammars to reflect a particular target, but I've never sat down and put in the work for that. Which is another way of saying that I'd be very interested in your ideas on how to do that, and that would be something really cool to incorporate back into the core PCFG code :)

**Reply by Legoclones** [reply]
Thanks!! I'll definitely keep you in the loop once I also find the time to invest in it

**Comment by bhavishya112** [comment]
hi Iakiw,
recently read your doc which implied that your pcfg cracker doesn't support gpu compute,
i might fail, but should i go up with adding this feature in your python based implementation?
i'd be using a cross platform framework like OpenCL
